### PR TITLE
Integrate disco plot code

### PR DESCRIPTION
**Pull request by aacic**

### CODE REVIEW

File: client/plots/disco/events.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
+import { tooltip } from '#src/client'
```
Comment (xzhou82) [inline client/plots/disco/events.js:1]: see if you can replace tooltip with `import {Menu} from '#dom/menu`

File: client/plots/disco/template.controls.js [hunk 2]
```diff
@@ -0,0 +1,617 @@
+import { Menu } from '#src/client'
```
Comment (xzhou82) [inline client/plots/disco/template.controls.js:1]: change to importing from '#dom/menu'

File: client/plots/disco/dt.disco.js [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+import { sayerror, hideerrors } from '#src/client'
+
+// these are shared across instances
+const mlabel = {}
+for (var key in mclass) {
+	mlabel[mclass[key].label] = mclass[key]
+	mlabel[key] = mclass[key]
+}
+
+const fusionColocationColors = {
```
Comment (xzhou82) [inline client/plots/disco/dt.disco.js:33]: this is defined in multiple scripts

Comment (aacic) [reply client/plots/disco/dt.disco.js]: I've removed it from this script. It was not used.

File: client/plots/disco/events.js (hunk 1 above)
Comment (aacic) [reply client/plots/disco/events.js:1]: I've tried to replace it with Menu, but the tooltip has additional methods like: `getPosChange`. I would say we keep it for now and I'll remove it when I refactor the whole disco plot. I can add a TODO for now.

Comment (aacic) [reply client/plots/disco/events.js]: Disco plot-related code is extracted to the helper.js script in the disco folder.

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**